tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -209,6 +219,43 @@ task HardFilterVcf {
   }
 }
 
+# This hard filtering matches DRAGEN 3.4.12. For later DRAGEN versions, this needs to be updated.
+task DragenHardFilterVcf {
```

Comment:
Should this be included in the DragenTasks wdl? (I'm fine with whatever others think is most appropriate, just wanted to raise the question)

Reply:
Fine either way, up for Lantern to decide I would say

Reply:
I would say, if we decide to move the alignment task to DragenTasks and keep all the dragen code together, let's move this one too. If not, it makes sense for the dragen version to live next to the non-dragen equivalent